tests/smoke-tests/README.md

```diff
@@ -0,0 +1,48 @@
+# Smoke Tests
+
+The purpose of the smoke tests is to find obvious issues in the e2e process of using the SAP Cloud SDK for AI libs.
+
+The sample code app is running on Cloud Foundry and uses the current canary version of the SAP Cloud SDK for AI.
```

Comment:
```suggestion
The sample application runs on Cloud Foundry and uses the current canary version of the SAP Cloud SDK for AI.
```

Reply:
(I believe `is running` is better to express that it is running all the time)